hamlpy/test/test_nodes.py

```diff
@@ -84,30 +97,17 @@ def test_inserts_nodes_into_proper_tree_depth(self):
         self.assertEqual(another_one_indentation_node, no_indentation_node.children[1])
 
     def test_adds_multiple_nodes_to_one(self):
-        start = self._create_node('%div')
-        one = self._create_node('  %div')
-        two = self._create_node('  %div')
-        three = self._create_node('  %div')
+        start = self._read_node('%div')
+        one = self._read_node('  %div')
+        two = self._read_node('  %div')
+        three = self._read_node('  %div')
 
         start.add_node(one)
         start.add_node(two)
         start.add_node(three)
 
         self.assertEqual(3, len(start.children))
 
-    def test_node_parent_function(self):
```

Comment:
`parent_of` was being used to figure out of a node belonged to filter node that no longer happens and this is no longer needed